test/realmodel/src/generateCase.js

```diff
@@ -71,26 +37,14 @@ if (matchFlatmodel) {
 async function saveToLocalFile(input) {
   let output = input.toString();
   let dataString;
-  let regexmodelname = /resnet/;
-  let matchFlatname = regexmodelname.test(`${JSON_DATA.getModelName()}`);
-  if (matchFlatname) {
+  if (matchFlatmodel) {
     input = parseInt(input);
-    if (buf.operation[input]) {
-      dataString = buf.operation[input];
-    } else if (buf1.operands[input]) {
-      dataString = buf1.operands[input];
-    } else if (buf2.operands[input]) {
-      dataString = buf2.operands[input];
-    } else if (buf3.operands[input]) {
-      dataString = buf3.operands[input];
-    } else if (buf4.operands[input]) {
-      dataString = buf4.operands[input];
-    } else if (buf5.operands[input]) {
-      dataString = buf5.operands[input];
-    } else if (buf6.operands[input]) {
-      dataString = buf6.operands[input];
-    } else {
-      throw ('please check input data');
+    for (i = 1; i < 7; i++) {
+      if (arr[0].operation[input]) {
```

Comment:
@shenx3x Please move this if (arr[0].operation[input]) block outside of for block